dev-python/setuptools/setuptools-67.8.0.recipe

```diff
@@ -70,14 +54,13 @@ INSTALL()
 		installLocation=$prefix/lib/$python/vendor-packages/
 		export PYTHONPATH=$installLocation:$PYTHONPATH
 		mkdir -p $installLocation
-
-		mv build$pythonVersion build
+		rm -rf build
 		export SETUPTOOLS_USE_DISTUTILS=local
-		$python setup.py install \
+#		$python setup.py egg_info
```

Comment:
Think this can be removed, the outcome is the same with or without.

Reply:
Removed ...

Reply:
Needs checking then :)

Reply:
Switched to pythonhosted for the file, the egg file's version looks correct now, doesn't need the line though.
It also fixes some other packages in the pipeline where the egg into isn't really correct (using pythonhosted link).
Thanks!